docs/source/models/gbd2017/risk_attributable_causes/iron_deficiency/index.rst

```diff
@@ -97,40 +140,33 @@ The sequela associated with the dietary iron deficiency cause in GBD 2017
 include mild iron deficiency anemia, moderate iron deficiency anemia, and 
 severe iron deficiency anemia. The severity of iron deficiency anemia is 
 determined by the WHO age- and sex- specific hemoglobin concentrations, as 
-described in the table below.
+described in the table above.
 
-.. list-table:: Hemoglobin Thresholds (g/L)
-   :widths: 15, 15, 15, 15
-   :header-rows: 1
+Iron Deficiency Risk Factor
++++++++++++++++++++++++++++
 
-	* - Group
-	  - Mild Anemia
-	  - Moderate Anemia
-	  - Severe Anemia
-	* - Males and Females <1 month
-	  - 130-149
-	  - 90-129
-	  - <90
-	* - Males and Females 1 month - 5 years
-	  - 100-109
-	  - 70-99
-	  - <70
-	* - Males and Females 5-14 years
-	  - 110-114
-	  - 80-109
-	  - <80
-	* - Males 15+ years
-	  - 110-129
-	  - 80-109
-	  - <80
-	* - Females 15+ years, non-pregnant 
-	  - 110-119
-	  - 80-109
-	  - <80
-	* - Females 15+ years, pregnant
-	  - 100-109
-	  - 70-00
-	  - <70
+The iron deficiency risk factor in GBD 2017 is a **continuous measure** modeled 
+as a distribution of hemoglobin concentrations among the population with 
+conditions that cause iron deficiency anemia. Below is a list of such causes: 
```

Comment:
We should probably list the cause id's for all these causes, but perhaps that belongs in a later section (e.g. the data section?) where where we go into more detail about how to model this cause in Vivarium.

Reply:
I added a table that goes through each of these lists, but I unfortunately had issues getting my table to render again. If anyone notices an issue in the code here please let me know!

Reply:
Ah, I see that it is an indentation issue again... I am working on updating this now

Reply:
Table is good to go now!